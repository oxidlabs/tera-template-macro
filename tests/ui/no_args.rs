use tera_template_macro::TeraTemplate;

#[derive(TeraTemplate, serde::Serialize)]
#[template]
struct Smoke {
    hello: String
}

fn main() {}