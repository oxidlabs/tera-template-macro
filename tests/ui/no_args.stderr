error: Expected attribute argument `path` in parentheses: `#[template(path = "index.html")]`
 --> tests/ui/no_args.rs:4:1
  |
4 | #[template]
  | ^^^^^^^^^^^
